lib/fetch.js

```diff
@@ -42,7 +30,11 @@ export function sanitizeEventAndNews(item, type) {
     strapline: item.data[`${type}.strapline`] ?
       item.data[`${type}.strapline`].value : null,
     datetime: item.data[`${type}.timestamp`] ?
-      mapDate(item.data[`${type}.timestamp`].value) : null,
+      item.data[`${type}.timestamp`].value : null,
```

Comment:
Wait no, it was the weird data structure from Prismic where property name contains a dot in it.